astropy/coordinates/baseframe.py

```diff
@@ -286,6 +275,25 @@ def __init_subclass__(cls, **kwargs):
         cls._create_readonly_property('frame_specific_representation_info',
                                       copy.deepcopy(repr_info))
 
+        # Set the frame attributes
+        # TODO: Should this be made to use readonly_prop_factory as well or
```

Comment:
The current functionality allows overriding `.frame_attributes`. Not that it's a good idea to allow this, but I think it's fine to leave functionality as is for now and reconsider this in a separate issue/PR.